core/modules/requestsMonitor/requestsMonitor.js

```diff
@@ -106,6 +106,7 @@ function addContentType(headerValue, entry) {
       break;
 
     case "video/webm":
+    case "video/mp4":
```

Comment:
I'm wondering if we can generate a short and small mp4 file (and `webm` too) and is it in integration tests.